lib/ansible/modules/source_control/gitlab_deploy_key.py

```diff
@@ -60,6 +60,8 @@
     default: present
     choices: [ "present", "absent" ]
 author: "Marcus Watkins (@marwatk)"
+requirements:
+    - python-gitlab python module
```

Comment:
gitlab_deploy_key doesn't have any dependencies, some of the other gitlab modules do, however.